README.md

```diff
@@ -20,98 +17,26 @@ To run this CLI application
-
-- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
-- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
-
-3. On a second terminal, deploy the test contract:
-
-```
-yarn deploy
-```
-
-This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
-
-- Hardhat => `packages/hardhat/contracts`
-- Foundry => `packages/foundry/contracts`
-
-The `yarn deploy` command uses a deploy script to deploy the contract to the network. You can customize it. Is located in:
-
-- Hardhat => `packages/hardhat/deploy`
-- Foundry => `packages/foundry/script`
-
-4. On a third terminal, start your NextJS app:
-
-```
-yarn start
-```
-
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
-
-Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
-
-**What's next**:
-
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
-- Edit your deployment scripts in `packages/hardhat/deploy`
-- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`
-
-## Documentation
-
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
-
-To know more about its features, check out our [website](https://scaffoldeth.io).
-
-## Contributing to Scaffold-ETH 2
-
-We welcome contributions to Scaffold-ETH 2!
-
-Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.
+## MVP
+- ETT CLI enables users to see the full scope of the tech tree
+- They can see clearly which ones are unlocked and which ones are still locked
+- It only allows them to interact with the challenges that are unlocked
+- It shows them their progress/general score on each challenge and total score on each branch of the tree
+- They can download a challenge locally
+- They can deploy and verify their contract with very few steps
+- They can submit the contract address for the challenge and get real-time feedback
+- If they pass the challenge it unlocks the next set of challenges
+- There are 5 - 10 challenges available. We will limit to a few branches at the start
+- CLI will show leaderboard
```

Comment:
If this was stitched with the builder profiles, we could envision a tab showing this leaderboard there, just sayin'

Reply:
Love the leaderboard idea. Friendly competition within the buidl guidl could be a huge driver for adoption and growth. We could eventually rank by # of challenges passed, # of passing tests for all challenges, total gas consumed by the contract submitted, etc..

Reply:
I'll add that to the future section. Definitely has been a thought that we would make this all visible inside BG app.